backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -1075,7 +1075,10 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
               val executionHandle = Future.successful(FailedNonRetryableExecutionHandle(StderrNonEmpty(jobDescriptor.key.tag, stderrSize, stderrAsOption), Option(returnCodeAsInt)))
               retryElseFail(status, executionHandle)
             case Success(returnCodeAsInt) if isAbort(returnCodeAsInt) =>
-              Future.successful(AbortedExecutionHandle)
+              val executionHandle = FailedNonRetryableExecutionHandle(StderrNonEmpty(jobDescriptor.key.tag, stderrSize, stderrAsOption), Option(returnCodeAsInt))
+              if (jobDescriptor.workflowDescriptor.workflowOptions.retryAbortedJobs)
+                retryElseFail(status, Future.successful(executionHandle))
```

Comment:
This means that if (and only if) `retryAbortedJobs` is enabled and the `maxRetries` runtime attribute is set to > 0, any aborted job (even user triggered) will be retried, which is not desired.
I think this should differentiate between "the backend unexpectedly killed this job and we want to retry it" and "the user aborted the workflow".
I'm wondering if the config backend should not send back a `FailedRetryableExecutionHandle` instead of writing a made up return code and returning "Failed"

Reply:
@ffinfo just pinging you to see you've had a chance to see the comment above and get your thoughts on it. Thank you!